ci/lint/04_install.sh

```diff
@@ -7,14 +7,25 @@
 export LC_ALL=C
 
 ${CI_RETRY_EXE} apt-get update
-${CI_RETRY_EXE} apt-get install -y python3-pip curl git gawk jq
-(
-  # Temporary workaround for https://github.com/bitcoin/bitcoin/pull/26130#issuecomment-1260499544
-  # Can be removed once the underlying image is bumped to something that includes git2.34 or later
-  sed -i -e 's/bionic/jammy/g' /etc/apt/sources.list
-  ${CI_RETRY_EXE} apt-get update
-  ${CI_RETRY_EXE} apt-get install -y --reinstall git
-)
+${CI_RETRY_EXE} apt-get install -y curl git gawk jq xz-utils
```

Comment:
Why add `xz-utils` here when it is only needed when compiling python?

Reply:
It is required here: https://github.com/bitcoin/bitcoin/blob/01ec5308bf616740c804247043046b23b483df5c/ci/lint/04_install.sh#L37